js/plates-view/planet-view.ts

```diff
@@ -7,14 +7,15 @@ import ForceArrow from "./force-arrow";
 import CrossSectionMarkers from "./cross-section-markers";
 import NPoleLabel from "./n-pole-label";
 import LatLongLines from "./lat-long-lines";
-import { rgbToHex, topoColor } from "../colormaps";
+import { topoColor } from "../colors/topographic-colors";
+import { RGBAFloatToHexNumber } from "../colors/utils";
 import getThreeJSRenderer from "../get-threejs-renderer";
 
 import "../../css/planet-view.less";
 
 // Mantle color is actually blue, as it's visible where two plates are diverging.
 // This crack should represent oceanic ridge.
-const MANTLE_COLOR = rgbToHex(topoColor(0.40));
+const MANTLE_COLOROR = RGBAFloatToHexNumber(topoColor(0.40));
```

Comment:
```suggestion
const MANTLE_COLOR = RGBAFloatToHexNumber(topoColor(0.40));
```
May have overshot the mark just a smidge. :wink: